MCServer/crafting.txt

```diff
@@ -76,8 +76,8 @@ NetherBrick             = netherbrickitem, 1:1, 1:2, 2:1, 2:2
 Glowstone               = GlowstoneDust, 1:1, 1:2, 2:1, 2:2
 Wool                    = String,    1:1, 1:2, 2:1, 2:2
 TNT                     = Gunpowder, 1:1, 3:1, 2:2, 1:3, 3:3 | Sand, 2:1, 1:2, 3:2, 2:3
-PillarQuartzBlock       = QuartzSlab, 1:1, 1:2
-ChiseledQuartzBlock, 2  = QuartzBlock, 1:1, 1:2
+PillarQuartzBlock, 2    = QuartzBlock, 1:1, 1:2
```

Comment:
Yeah. You stack 2 normal quartz blocks to get the pillar.
To get the chiseled quartz block, you stack 2 quartz slabs.